roles/k3s/tasks/config-neutron.yml

```diff
@@ -0,0 +1,51 @@
+---
+- name: Create calico network
+  kolla_toolbox:
+    module_name: os_network
+    module_args:
+      auth: "{{ openstack_auth }}"
+      name: caliconet
+      provider_network_type: flat
+      provider_physical_network: calico
+      shared: yes
+      state: present
+  run_once: true
+  become: yes
+
+- name: Create calico subnet
+  kolla_toolbox:
+    module_name: os_subnet
+    module_args:
+      auth: "{{ openstack_auth }}"
+      network_name: caliconet
+      name: caliconet-subnet
+      cidr: "{{ k3s_cluster_cidr }}"
+      enable_dhcp: no
+  become: yes
+
+# FIXME(jason): this doesn't actually add the subnet to the router!
```

Comment:
Just highlighting this FIXME, maybe in Xena it's fixable b/c Ansible is at a later version.

Reply:
I actually have a fix for this in some later commits! 

* [subnets_info](https://docs.ansible.com/ansible/latest/collections/openstack/cloud/subnets_info_module.html#ansible-collections-openstack-cloud-subnets-info-module)
* [subnet](https://docs.ansible.com/ansible/latest/collections/openstack/cloud/subnet_module.html#ansible-collections-openstack-cloud-subnet-module)
* [routers_info](https://docs.ansible.com/ansible/latest/collections/openstack/cloud/routers_info_module.html#ansible-collections-openstack-cloud-routers-info-module)
* [routers](https://docs.ansible.com/ansible/latest/collections/openstack/cloud/router_module.html#ansible-collections-openstack-cloud-router-module)